libcontainer/container_linux.go

```diff
@@ -482,6 +482,7 @@ func (c *linuxContainer) newParentProcess(p *Process) (parentProcess, error) {
 }
 
 func (c *linuxContainer) commandTemplate(p *Process, childInitPipe *os.File, childLogPipe *os.File) *exec.Cmd {
+	// #nosec: G204 (Subprocess launched with a potential tainted input or cmd arguments)
 	cmd := exec.Command(c.initPath, c.initArgs[1:]...)
```

Comment:
I should probably add here that, when using golangci-lint, the comments are formatted different; something like

```
//nolint:gosec
```

or 

```
//nolint:gosec // optional comment describing why it's excluded
```

If possible (doesn't work in _all_ cases), I generally prefer the comment to be on the same line as the actual code, e.g.;

```suggestion
	cmd := exec.Command(c.initPath, c.initArgs[1:]...) //nolint:gosec // ignore G204 (Subprocess launched with a potential tainted input or cmd arguments)
```

Reply:
Imo the nosec comments should not go away, because otherwise if you run `gosec` (or `make gosec`), they would not apply. Also, are you sure that they have to be different? Because they seem to work, otherwise the action would not be green.

Reply:
I'd rather see `//nolint:gosec` annotations, if they work.

Reply:
Done